packages/ComponentLibrary/src/components/Drawer/DrawerSection/index.tsx

```diff
@@ -40,13 +40,12 @@ const DrawerSection = ({ item, onClick }: DrawerSectionProps) => {
         selected={isSelected}
         expanded={expanded}
       />
-
-      {item.type === MenuType.Folder && item.submenu ? (
-        <Collapse in={expanded} timeout="auto" unmountOnExit>
+      {item.children ? (
```

Comment:
I think is better to use && instead. Or if u want to use '?'  add an else return null